modules/gapi/src/api/gcall_priv.hpp

```diff
@@ -42,10 +42,11 @@ class GCall::Priv
 {
 public:
     std::vector<GArg> m_args;
-    const GKernel     m_k;
+    GKernel     m_k;
```

Comment:
I'd add a comment here why it is no longer `const` anymore

Reply:
Fixed